cli/Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -722,6 +723,8 @@ extension ProjectDescription.Product {
             return .commandLineTool
         case .test:
             return .unitTests
+        case .plugin:
+            return .commandLineTool // MARK: plugin must be added to Product.swift?
```

Comment:
I was thinking we could introduce a new type, `.plugin` or `.swiftpmPlugin` or adopt some naming convention such that we can detect it at generation time and generate a Scheme to make it convenient to run the plugin through Xcode. Thoughts @tuist/engineering ?